recipe/meta.yaml

```diff
@@ -17,7 +17,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
+  skip: True  # [unix and not py35]
```

Comment:
Test suite is built at the same time the library is built, so I cannot build the tests in a later stage.